airbyte-integrations/connectors/source-aha/source_aha/aha.yaml

```diff
@@ -0,0 +1,54 @@
+version: "0.1.0"
+
+definitions:
+  selector:
+    extractor:
+      field_pointer: []
+  requester:
+    url_base: "{{ config['url'] }}"
+    http_method: "GET"
+    authenticator:
+      type: "BearerAuthenticator"
+      api_token: "{{ config['api_key'] }}"
+  increment_paginator:
+    type: DefaultPaginator
+    url_base: "*ref(definitions.requester.url_base)"
+    page_size_option:
+      inject_into: "request_parameter"
+      field_name: "limit"
+    pagination_strategy:
+      type: PageIncrement
+      page_size: 5
+    page_token_option:
+      inject_into: "request_parameter"
+      field_name: "page"
+  retriever:
+    record_selector:
+      $ref: "*ref(definitions.selector)"
+    paginator:
+      type: NoPagination
+    requester:
+      $ref: "*ref(definitions.requester)"
+
+  base_stream:
+    retriever:
+      $ref: "*ref(definitions.retriever)"
+  features_stream:
+    $ref: "*ref(definitions.base_stream)"
+    $options:
+      name: "features"
+      path: "/api/v1/features"
```

Comment:
Please move the prefix `/api/v1` to base URL.

Reply:
Moved it to base URL